safe-global/web-core
# fix: adjust number formatters + "Max" spending limit

## What it solves

Number formatting and "Max" spending limit.

## How this PR fixes it

The number formatters have been updated to use `ethers`' `formatUnits` instead of `toDecimals` and the "Max" token amount has been removed from the `<SpendingLimitForm>` field as it's possible to set higher amounts than that which is in the Safe.

## How to test it

1. Observe all numbers formatted correctly across the Safe, i.e. in the assets list, in high and low amount transactions and in modals. Choosing the "Max" amount when the current transaction has high or low amounts, formatted with `>` or `<` should not copy these symbols into their respective amount fields.
2. The "Max" button should no longer exist in the spending limits form.
